cmd/kubeadm/app/constants/constants.go

```diff
@@ -303,6 +303,9 @@ const (
 	// YAMLDocumentSeparator is the separator for YAML documents
 	// TODO: Find a better place for this constant
 	YAMLDocumentSeparator = "---\n"
+
+	// DefaultKubeConfig is the default localtion of KubeConfig
+	DefaultKubeConfig = "/etc/kubernetes/admin.conf"
```

Comment:
`/etc/kubernetes/admin.conf` is used i a lot of places:
https://github.com/kubernetes/kubernetes/search?p=1&q=%22%2Fetc%2Fkubernetes%2Fadmin%22&unscoped_q=%22%2Fetc%2Fkubernetes%2Fadmin%22

better use this as the github search is sometimes wrong:
```
cd cmd/kubeadm
grep "/etc/kubernetes/admin.conf" * -rnI
```
we need to use the constant everywhere.

Reply:
@neolit123 Yes, I just missed that. Thank you for your catch.